Packs/HatchingTriage/Integrations/HatchingTriage/HatchingTriage.yml

```diff
@@ -182,7 +191,7 @@ script:
   - arguments:
     - default: false
       description: Sample's unique identifier, can be found using the query samples command
```

Comment:
Please write here in the description that the input can be multiple IDs as comma separated